lib/OptimizationNLPModels/test/runtests.jl

```diff
@@ -0,0 +1,16 @@
+using OptimizationNLPModels, Optimization, NLPModelsTest
+using Test
+
+@testset "NLPModels" begin
+    # Problem 5 in the Hock-Schittkowski suite
+    hs5f(u, p) = sin(x[1] + x[2]) + (x[1] - x[2])^2 - (3 / 2) * x[1] + (5 / 2)x[2] + 1
+    f = Optimization.OptimizationFunction((x, p) -> hs5f(u, p), Optimization.AutoZygote())
+    lb = [-1.5; -3]
+    ub = [4; 3]
+    u0 = [0.0; 0.0]
+    oprob = Optimization.OptimizationProblem(
+        f, u0, lb = lb, ub = ub, sense = Optimization.MinSense)
+    nlpmo = NLPModelsTest.HS5()
+
+    converted = OptimizationNLPModels.OptimizationProblem(nlpmo)
```

Comment:
You can test by running it with a solver to check they give the same solutions

Reply:
Sounds like a good idea. I was also considering comparing the counters (number of calls to the objective function, constraint function, etc...) since they're already available on the `NLPModel` interface. Do you think that would be a good idea? I suppose that would be very implementation-dependent and therefore not very useful. On the other hand, should I stick to a solver already included in base Optimization.jl for comparing solutions?

Reply:
We could compare the counters but I am not sure how useful that'd be. Yeah let's use the LBFGS available in Optimization.jl for start to get it running

Reply:
That can handle constraints too